src/gui/MainWindow.cpp

```diff
@@ -487,6 +494,7 @@ void MainWindow::initGui()
     menu->addAction(mToolBar->saveAction());
     menu->addAction(mSaveAsAction);
     menu->addAction(mUploadAction);
+    menu->addAction(mCopyToClipboardAction);
```

Comment:
The other copy action is under `Edit`, maybe this one would be better suited there just under the existing copy action.